src/test/java/uk/gov/hmcts/reform/sscs/reference/data/service/VerbalLanguagesServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.sscs.reference.data.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sscs.reference.data.helper.ReferenceDataHelper.getDuplicates;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import uk.gov.hmcts.reform.sscs.reference.data.model.Language;
+
+public class VerbalLanguagesServiceTest {
+
+    VerbalLanguagesService verbalLanguages;
+
+    @Before
+    public void setup() {
+        verbalLanguages = new VerbalLanguagesService();
+    }
+
+    @DisplayName("There should be no duplicate Verbal Languages")
+    @Test
+    public void testVerbalLanguagesDuplicates() {

```

Comment:
Worth while adding a parametrised test to test all the different languages passed though from SYA? Might be over testing but I guess it ensure non of the values are null or wrong